From 03402e430cdfe6e8469f0d2fc9cd29a13e472fd9 Mon Sep 17 00:00:00 2001
From: Michael Ray <michael.ray@arm.com>
Date: Tue, 21 Jan 2020 12:32:48 -0600
Subject: [PATCH] [PATCH] Fake devicedb running on local machine

Current way of running devicedb is within docker and therefore
devicedb is not running as a local process
---
 networking/manager.go | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/networking/manager.go b/networking/manager.go
index 2f3ece2..fceae1f 100644
--- a/networking/manager.go
+++ b/networking/manager.go
@@ -49,7 +49,6 @@ import (
 	"github.com/armPelionEdge/maestro/log"
 	"github.com/armPelionEdge/maestro/maestroConfig"
 	"github.com/armPelionEdge/maestro/networking/arp"
-	"github.com/armPelionEdge/maestro/processes"
 	"github.com/armPelionEdge/maestro/storage"
 	"github.com/armPelionEdge/maestro/tasks"
 	"github.com/armPelionEdge/maestroSpecs"
@@ -912,7 +911,8 @@ func (this *networkManagerInstance) initDeviceDBConfig() {
 		log.MaestroInfof("initDeviceDBConfig: Waiting for devicedb process/job\n")
 		for totalWaitTime < MAX_DEVICEDB_WAIT_TIME_IN_SECS {
 			//First wait for devicedb to start
-			devicedbrunning, pid = processes.IsJobActive(DEVICEDB_JOB_NAME)
+			devicedbrunning = true
+			pid = 8573
 			log.MaestroWarnf("initDeviceDBConfig: devicedbrunning: %v, pid: %d\n", devicedbrunning, pid)
 			if devicedbrunning {
 				//Service is started, but wait for some seconds for the port to be up and running
-- 
2.21.0

